### PR TITLE
added jsx-a11y and added some a11y updates to the site

### PR DESCRIPTION
**Pull request by johnnyxbell**

### CODE REVIEW

File: app/components/Header.js [hunk 1]
```diff
@@ -167,10 +173,10 @@ const Navigation = ({ onMenuClick = () => {} }) => {
         </ul>
       </section>
       <section className="Header__Nav__Social">
-        <Icon href="https://twitter.com/ReactRally" type="twitter" />
-        <Icon href="https://github.com/react-rally" type="github" />
-        <Icon href="https://instagram.com/reactrally" type="instagram" />
-        <Button href={constants.Links.TICKET_SALES} className="medium">
+        <Icon href="https://twitter.com/ReactRally" type="React Rally Twitter" />
```
Comment (mzabriskie) [inline app/components/Header.js]: The `<Icon />` component expects `type` prop to be something that can be used for it's icon resource. I would suggest keeping `type` as it was, and internal to `<Icon />` add `aria-label="React Rally ${type}"`.

File: app/screens/Schedule/index.js [hunk 2]
```diff
@@ -101,22 +101,22 @@ export default class extends React.Component {
   renderMenu() {
     return (
       <menu className="Schedule__Menu">
-        <a
+        <button
           href="javascript://"
```
Comment (mzabriskie) [inline app/screens/Schedule/index.js]: The `href` prop is no longer needed.

**Review by mzabriskie — CHANGES_REQUESTED** [review]
Couple changes suggested as well as a rebase required to merge.

Thanks for the PR!

**Comment by johnnyxbell** [comment]
Old PR no longer required.

**Pull request closed without merging**